docs/beast_development.rst

```diff
@@ -4,25 +4,33 @@
 BEAST Development
 #################
 
-You are encouraged to help maintain and improve the BEAST. Before doing so,
+Basic Workflow
+==============
+
+You are encouraged to help maintain and improve the ``beast``. Before doing so,
 please familiarize yourself with basic version control and Git workflow
-concepts using one or more of these guides:
+concepts. A good description of the standard fork-branch-pull workflow can be
+found here:
+
+- https://gist.github.com/Chaser324/ce0505fbed06b947d962
+
+Some other useful guides are:
 
 - https://guides.github.com/introduction/flow/
 - https://lifehacker.com/5983680/how-the-heck-do-i-use-github
 - https://homes.cs.washington.edu/~mernst/advice/version-control.html
 - https://www.youtube.com/watch?v=y_YKHXuJ-ak
 
-Here is the recommended work-flow for contributing to the BEAST project.
-Details follow.
+In short, the fork-branch-pull workflow for contributing to the ``beast``
+project is:
 
-- Create your own 'fork' of the official BEAST release
+- Create your own 'fork' of the official ``beast`` release

```

Comment:
Instead of 'release', maybe 'repository' is more appropriate.  We have releases now.